app/serializers/spree/v2/storefront/vendor_serializer.rb

```diff
@@ -4,7 +4,10 @@ module Storefront
       class VendorSerializer < BaseSerializer
         set_type :vendor
 
-        attributes :name, :about_us
+        attributes :name, :about_us, :slug, :state, :contact_us
```

Comment:
@poudelprakash let's not include state here

Reply:
@damianlegawiec Done, State has been removed